ia/test/unitTest.py

```
@@ -0,0 +1,29 @@
+
+
+class SimpleEncodingTestCase(unittest.TestCase):
+    def test_encrypt_basic(self):
+        test_strings = ["hello", "", "a", "123"]
+
+        for test_str in test_strings:
+            encrypted = msg.encrypt(test_str)
+            decrypted = msg.decrypt(encrypted)
+            self.assertEqual(test_str, decrypted, f"Failed for input: '{test_str}'")
+
+    def test_encrypt_decrypt_inverse(self):
```

Ruff (0.11.9)

Missing return type annotation for public function `test_encrypt_decrypt_inverse`

Add return type annotation: `None`

(ANN201)